tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/documentdashboard.controller.js

```diff
@@ -158,7 +162,7 @@
                     });
                 }
             });
-
+            //Declaring column collection object.

```

Comment:
Developers can understand its a column collection by seeing code. Comment should tell why you are defining the collection.